data/EN/Event/2023/03/2023-03.json

```diff
@@ -840,11 +840,14 @@
       },
       {
         "index": 2,
-        "url": ">https://www.youtube.com/c/GenshinImpact",
-        "urlOriginal": ">https://www.youtube.com/c/GenshinImpact",
-        "urlOriginalResolved": [],
+        "url": "https://www.youtube.com/c/GenshinImpact",
+        "urlOriginal": "https://www.youtube.com/c/GenshinImpact",
+        "urlOriginalResolved": [
+          "https://www.youtube.com/c/GenshinImpact",
+          ""
```

Comment:
Why is there an empty url? What is urlOriginalResolved for?

Reply:
I'll need to write some README to explain a bit better what each thing is for.

I chose to have an empty string at the end to show that it was successfully resolved into a destination.

If you have a URL that would end with a non-resolved destination (final redirection was removed or something, 404 whatever), you'll get no `""`. At least that's the idea.
So whoever will use the data can decide whether they want to jump to the resolved URL (better perf) or to use the original one.

There are some things to be improved though. Some Youtube links are not perfect (this one here is perfect), as YT can sometimes redirect to show the General Conditions or something. But because their links are pretty standards, we could have some hardcoded redirections to skip a lot of poking.

Reply:
Created follow-up Issue #40 for README creation.

Reply:
I think being unable to resolve an url should lead to an empty array instead.
Otherwise you need to parse each array and check for the empty string instead of just the length.

Also im not quite sure why we would use urlOriginal if urlOriginalResolved has no drawbacks and all urls until now are fully resolved. In the worst case they match and in the best case the directly link to source without redirects.

Reply:
Also why is urlOriginalResolved an Array? Does it contain every redirect between urlOriginal and source?
If not maybe switching its type to string | undefined would suite it better while the resolved state can be reflected by an enum.

Reply:
Yes, it contains every redirect indeed.

Reply:
This has been discussed and has been changed in https://github.com/dval-in/dvalin-data/pull/42